scripts/tests/twister_blackbox/test_footprint.py

```diff
@@ -103,11 +108,11 @@ def test_compare_report(self, caplog, out_path, old_ram_multiplier, expect_delta
 
         if expect_delta_log:
             assert self.RAM_KEY in caplog.text
-            assert self.DELTA_WARNING_RELEASE in caplog.text, \
+            assert re.search(self.DELTA_WARNING_COMPARE, caplog.text), \
```

Comment:
just qeustion can we use the fuzzywuzzy to match the string for all tests? this will save a lot of pattern formation code

Reply:
Fuzzy match is usually good for deduplication or anomaly detection. I think it is less applicable for 'blackbox' tests here as the distance metric might hide some essential change and its pass/fail threshold also needs to be chosen somehow.